keyboards/handwired/dactyl_manuform/5x8/keyboard.json

```diff
@@ -0,0 +1,145 @@
+{
+    "keyboard_name": "Dactyl-Manuform (5x8)",
+    "manufacturer": "tshort",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x444D",
+        "pid": "0x3537",
+        "device_version": "0.0.1"
+    },
+    "rgblight": {
+        "led_count": 12
+    },
+    "ws2812": {
+        "pin": "D3"
+    },
+    "features": {
+        "bootmagic": false,
+        "command": true,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "matrix_pins": {
+        "cols": ["B6", "B2", "B3", "B1", "F7", "F6", "F5", "F4"],
+        "rows": ["D4", "C6", "D7", "E6", "B4", "B5"]
+    },
+    "diode_direction": "COL2ROW",
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D0"
```

Comment:
```suggestion
        "serial": {
            "pin": "D0"
        }
```

Reply:
"serial": {"pin": "D0"} not worked on my keyboard. "soft_serial_pin":"D0" worked

Reply:
Well not sure what could have gone wrong — your repo is up to date enough. I assure you, this is the correct way to specify the split communication pin. By "didn't work", do you mean didn't compile? Or didn't functionally work?

Reply:
I fixed this code. The keyboard functionality is working. Thanks